test/media/vtt_text_parser_unit.js

```diff
@@ -46,6 +47,9 @@ describe('VttTextParser', function() {
     verifyHelper([],
         'WEBVTT',
         { periodStart: 0, segmentStart: 0, segmentEnd: 0 });
+
+    // window.VTTCue is not overwritten
+    expect(window.VTTCue.toString()).toBe('function () {}');
```

Comment:
Although this should work in most cases, it's possible it wouldn't catch an instance where VTTCue is overwritten by another empty function. Instead maybe just store a reference to a function, set `window.VTTCue = funcReference` in `beforeEach`, then test that `window.VTTCue` is the same as the reference var in the test.